From 73c75ccb8eb20aa1514e8c86deb859e70bc63290 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Sat, 15 Apr 2023 10:20:47 +0100
Subject: [PATCH] Fix build with GCC 13

GCC 13 (as usual for new compiler releases) shuffles around some internal includes so some
are no longer transitively included.

See https://gnu.org/software/gcc/gcc-13/porting_to.html.

Bug: https://bugs.gentoo.org/894742
---
 common/sample_format.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/sample_format.hpp b/common/sample_format.hpp
index 900655f7..b68d00a3 100644
--- a/common/sample_format.hpp
+++ b/common/sample_format.hpp
@@ -19,6 +19,7 @@
 #ifndef SAMPLE_FORMAT_H
 #define SAMPLE_FORMAT_H
 
+#include <cstdint>
 #include <string>
 
 
